docs/how_to_contribute.rst

```diff
@@ -61,3 +61,24 @@ until all checks pass.
 
 
 Once you got everything to pass, you can stage and commit your changes and push them to the remote github repository.
+
+How to change the documentation
+-------------------------------
+
+In order to build the documentation locally you should build and activate the docs environment::
+
+   conda env create -f docs/environment.yml
```

Comment:
```suggestion
   mamba env create -f docs/environment.yml
```

We already recommend mamba elsewhere, why not here?

Reply:
Sure thing.